README.md

```diff
@@ -132,6 +132,15 @@ service HealthCheckService {
 
 ---
 
+## gRPC Status Codes
+
+The following gRPC Status Code must be handled by clients of the DAS Protocol:
+* `UNAVAILABLE`: When a DAS server is not available temporarily, which can occur during a rolling upgrade, the DAS client should retry the call for a period of time.
+* `UNAUTHENTICATED` or `PERMISSION_DENIED`: When a DAS server is not able to access a data source, it may return an unauthenticated or permission denied response, indicating that additional options configuration is needed.
```

Comment:
Still thinking about someone reading the paragraph and implementing a server.

Operations that aren't working in the backend, are in general leading to `INVALID_ARGUMENT` (right? Just checking.) Like "trying to insert a string too large in field X". However a server should return a different error code if the error in the backend is due to permission denied?